stroeer/web/article/v1/web_article_service.proto

```diff
@@ -17,10 +18,13 @@ service WebArticleService {
 
 message GetWebArticlePageRequest {
   string id = 1;
+  string page = 2;
 }
 
 message GetWebArticlePageResponse {
   WebArticle web_article = 1;
   repeated RelatedWebArticle related_web_articles = 2;
+  repeated stroeer.core.v1.BodyNode body_nodes = 3;
+  uint32 pages = 4;
```

Comment:
isnt `pages` the indicator, which page you got?
see https://cloud.google.com/apis/design/design_patterns#list_pagination

```
GetWebArticlePageRequest {
  string id = 1;
  int32 page_size = 2;
  string page_token = 3;
}
```

```
GetWebArticlePageResponse {
[...]
repeated BodyNode body_nodes = 3;
string next_page_token = 4; // can be empty if is the last
}
```


I would say we should talk about this with SEO Björn and Martin.


`/politik/ausland/2020-08/kamala-harris-usa-demokraten-parteitag-joe-biden-donald-trump/seite-2`
this is good for generating more clicks, but how would you handle this for old coremedia articles? have if blocks in the code to treat them differently?
how would you fetch the data to display it on one page? have sequential calls?